Elastiflix/docker-compose-elastic-otel.yml

```diff
@@ -26,7 +26,7 @@ services:
     networks:
       - app-network
     environment:
-      - TOGGLE_CLIENT_PAUSE=${TOGGLE_CLIENT_PAUSE}
+      - TOGGLE_CLIENT_PAUSE=${TOGGLE_CLIENT_PAUSE:-}
```

Comment:
[for reviewer] this prevents having a warning when the app starts and the variable it not set.